src/Hydration.js

```diff
@@ -1,30 +1,27 @@
 class Hydration {
-  constructor(data) {
-    this.data = data;
-    this.userHydrationData = [];
+  constructor(hydrationData) {
+    this.hydrationData = hydrationData;
```

Comment:
To confirm, hydrationData contains _all_ the user's hydration data?

Reply:
Yes the hydration contains all user's hydration data.